.changes/unreleased/Fixes-20230320-153225.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixes
```

Comment:
https://github.com/dbt-labs/dbt-bigquery/issues/615 is currently labeled as a feature / enhancement. If so, you'd want to update the `kind` accordingly.

Do you think it should be considered a bug instead?

Reply:
If you do change the `kind`, make sure that the `custom` area is updated accordingly. I believe bugs take an `Issue` whereas features take a `PR`. But check either way (which you could do by just generating a new changie log of the corresponding type).